plugins/fr_ch/src/src.pro

```diff
@@ -20,25 +20,25 @@ TARGET          = $$qtLibraryTarget(frplugin)
 EXAMPLE_FILES = frenchplugin.json
 
 # generate database for presage:
-PLUGIN_INSTALL_PATH = $${UBUNTU_KEYBOARD_LIB_DIR}/fr/
```

Comment:
Are ou removing french here?

Reply:
no, without this it will place it in the fr folder, we want them to be where the qml files are

Reply:
ah, sorry, nvm